README.md

```diff
@@ -27,4 +56,134 @@ See
 * [CRUDModel](api/src/main/java/dev/akif/crud/CRUDModel.java)
 * [CRUDEntity](api/src/main/java/dev/akif/crud/CRUDEntity.java)
 
-and their `Simple`, `Simpler` and `Simplest` variants.
+and their Simple, Simpler, and Simplest variants.
+
+## Code Examples
+
+### Writing a controller
+
+The code snippet represents a Java Controller that implements the CRUDController interface.
```

Comment:
`CRUDController` is an abstract class so this statement and following example would need to be updated.